lib/ringcaptcha/api.rb

```diff
@@ -35,5 +35,16 @@ def self.do_http_request(http, request)
         end
       end
     end
+
+    def self.service_up?
+      uri = URI.parse("https://api.ringcaptcha.com")
+      http = Net::HTTP.new(uri.host, uri.port)
+      http.use_ssl = true
+      http.verify_mode = OpenSSL::SSL::VERIFY_NONE
+
+      response = http.request(Net::HTTP::Post.new("/status"))
+
+      response.is_a?(Net::HTTPSuccess) && response.body == "WE ARE ALIVE ;-)"
```

Comment:
You're right, I should used GET. I'll change that.
Why not check the response body? They could for instance return 200 with a body of "we're down". I prefer to be more strict.